static/header.less

```diff
@@ -200,8 +198,8 @@
             background: white;
             .border;
             border-top: none;
-            right: -5px;
-            top: @gutter*3;
+            right: 0;
+            top: @brand-height - 1px;// -1 so there’s a little overlap
```

Comment:
I moved which element is `position: relative` so the `right` and `top` make more sense now.